src/js/Benchmarks/offline/suite_rms.js

```diff
@@ -52,6 +71,51 @@ export default function rms(essentia, Meyda, audioURL) {
             p.textContent = 'Fastest is ' + this.filter('fastest').map('name');
             RMSButton.classList.remove('is-loading');
             RMSButton.disable = false;
+
+            meyda_results.classList.remove('is-hidden');
+            essentia_results.classList.remove('is-hidden');
+
+            meyda_mean.textContent = this[0].stats.mean;
+            meyda_moe.textContent = this[0].stats.moe;
+            meyda_rme.textContent = this[0].stats.rme;
+            meyda_sem.textContent = this[0].stats.sem;
+            meyda_deviation.textContent = this[0].stats.deviation;
+            meyda_variance.textContent = this[0].stats.variance;
+
+            ess_mean.textContent = this[1].stats.mean;
+            ess_moe.textContent = this[1].stats.moe;
+            ess_rme.textContent = this[1].stats.rme;
+            ess_sem.textContent = this[1].stats.sem;
+            ess_deviation.textContent = this[1].stats.deviation;
+            ess_variance.textContent = this[1].stats.variance;
+
+            violinDistributionPlot(meyda_plot, {0:["meyda", this[0].stats.sample, "green"]}, "Time distribution RMS - Meyda");
+            violinDistributionPlot(ess_plot, {0:["essentia.js",this[1].stats.sample, "red"]}, "Time distribution RMS - Essentia");
+            violinDistributionPlot(stack_plot, {0:["meyda", this[0].stats.sample, "green"], 1:["essentia.js", this[1].stats.sample, "red"]},
+                                     "Time distribution RMS - Stack");
+
+            const resultsObj = {
+                "meyda": {
+                    "mean": this[0].stats.mean,
+                    "moe": this[0].stats.moe,
+                    "rme": this[0].stats.rme,
+                    "sem": this[0].stats.sem,
+                    "deviation": this[0].stats.deviation,
+                    "variance": this[0].stats.variance,
+                    "execution times": this[0].stats.sample
+                },
+                "essentia": {
+                    "mean": this[1].stats.mean,
+                    "moe": this[1].stats.moe,
+                    "rme": this[1].stats.rme,
+                    "sem": this[1].stats.sem,
+                    "deviation": this[1].stats.deviation,
+                    "variance": this[1].stats.variance,
+                    "execution times": this[1].stats.sample
+                }
+            }
+            downloadJson(resultsObj, "rms.json");
```

Comment:
could it be better to download when user click?

Reply:
At the moment I want it to work like it is now. Maybe in the future we can put a download button.